container-images/tcib/base/os/ironic-base/ironic-conductor/ironic-conductor.yaml

```diff
@@ -6,6 +6,7 @@ tcib_packages:
   - dosfstools
   - e2fsprogs
   - gdisk
+  - inotify-tools
```

Comment:
https://logserver.rdoproject.org/05/205/0795bb902af2199905413cae8593d382618459e8/github-check/openstack-meta-content-provider/d7ccfa5/ci-framework-data/logs/d3e43c59-c5fe-4fb2-8418-128c2311a824/base/os/ironic-base/ironic-conductor/ironic-conductor-build.log
```
Error: Unable to find a match: inotify-tools
```
Need to enable proper repo to install this package.